<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymbolsDlg</class>
 <widget class="QDialog" name="SymbolsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symbols</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="6" colspan="2">
    <widget class="QComboBox" name="comboFonts"/>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushSearch">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkCode">
     <property name="text">
      <string>Insert code</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeGroups">
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableSymbols">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineSymbols"/>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="lineSearch"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushCopy">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushAdd">
     <property name="text">
      <string>Add to QP</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
